includes/wikia/controllers/HeliosHelperController.class.php

```diff
@@ -118,8 +118,11 @@ private function emailControllerForTokenContext( string $tokenContext = null ):
 		if ( $tokenContext === 'facebook' ) {
 		    return 'Email\Controller\FacebookDisconnect';
         }
-        if ( $tokenContext === 'google' ) {
-		    return 'Email\Controller\GoogleDisconnect';
+		if ( $tokenContext === 'google' ) {
+			return 'Email\Controller\GoogleDisconnect';
+		}
+		if ( $tokenContext === 'twitch' ) {
+			return 'Email\Controller\GoogleDisconnect';
```

Comment:
Shouldn't this read TwitchDisconnect?

```suggestion
			return 'Email\Controller\TwitchDisconnect';
```